tools/aws_lab_setup/ansible.cfg

```diff
@@ -11,7 +11,7 @@
 
 # some basic default values...
 
-inventory      = ./inventory
+inventory      = ./inventory/all
```

Comment:
Does this really need to change? I realize the groups_vars that lived in here are gone that you only have the `all` file. Looking ahead to that file -- localhost is assumed by Ansible so this file and configuration directive is unnecessary now. Even if it where, I would have suggested we follow the convention of a simple `hosts` file with no directory.

Reply:
>localhost is assumed by Ansible
the reason this was added is otherwise the provisioner spews purple warnings, Ansible does not seem to like implying that without warnings.

Reply:
Fair enough. Here is a compromise to these changes -- point the inventory at `/dev/null`. That was a trick I'd use when engine required an inventory exist. They've cut it back to a warning the we have now and get why that is suboptimal.  Sound good?